assets/airports/kstl.json

```diff
@@ -0,0 +1,13310 @@
+    "ROD":   ["N40.28780185065528", "W84.04307555556447"],
+    "ROTRY": ["N39.37714352849269", "W90.21993255058317"],
+    "RRIPP": ["N38.70637449380956", "W90.48220824640052"],
+    "SAYDI": ["N37.84695285763814", "W88.37274169353302"],
+    "SCRCH": ["N37.07607057500629", "W89.48455809985319"],
+    "SEIZE": ["N40.67827737596616", "W91.90328978945894"],
+    "SEKAY": ["N38.62126963346221", "W90.48813628595165"],
+    "SEXTN": ["N39.84729436360059", "W90.63947295588310"],
+    "SGF":   ["N37.35596767764664", "W93.33407592236173"],
+    "SHB":   ["N39.63255747721583", "W85.82430266749328"],
+    "SKOVE": ["N39.77547311771913", "W91.21957396910330"],
+    "SKUTR": ["N40.06835084427763", "W90.06083678641298"],
+    "SNYDR": ["N38.71444722909383", "W90.52542113702823"],
+    "SPI":   ["N39.83975534084522", "W89.67775725758021"],
+    "SSKOT": ["N38.92309225929712", "W89.68670653736594"],
+    "STAAN": ["N38.79599853235449", "W90.87966155453105"],
+    "TATRR": ["N39.02067740924412", "W90.59298705500125"],
+    "TEDDD": ["N38.53661867002354", "W89.57510375415576"],
+    "TEEBE": ["N38.36018006153978", "W90.09475707450106"],
+    "TINGS": ["N37.30300625628679", "W88.25619506266628"],
+    "TRAKE": ["N38.69188772135863", "W91.74256896425348"],
+    "TRAVS": ["N38.60495913815178", "W90.35616301934176"],
+    "TWRAY": ["N37.60139383476628", "W91.01458739682472"],
+    "UKAAY": ["N39.17102619933542", "W88.87957763106500"],
+    "VEEGE": ["N39.14220341472765", "W90.78515624446650"],
+    "VHP":   ["N39.81470719132302", "W86.36750792875802"],
+    "VIH":   ["N38.15405120620728", "W91.70679473329419"],
+    "VISQA": ["N36.29040806334366", "W89.03512572677795"],
+    "WAARN": ["N38.76002677796801", "W90.54162597101363"],
+    "WESCO": ["N38.02132008115019", "W91.22325896666212"],
+    "WHRLI": ["N39.10097200504934", "W91.07494353696526"],
+    "WLDBL": ["N39.59242598354066", "W88.62753295331471"],
+    "WOLTR": ["N38.82815432808154", "W90.41748046319330"],
+    "WORKE": ["N39.11748877415269", "W87.77368926429600"],
+    "ZOLLI": ["N38.32005696683986", "W88.86964415938466"],
+    "ZULMO": ["N38.59422652686858", "W91.39615630554020"]
+  },
+  "runways":[
+    {
+      "name": ["6", "24"],
+      "end": [["N38d44m48.04", "W90d22m52.43", "551ft"], ["N38d45m22.38", "W90d21m27.02", "534ft"]],
+      "delay": [5, 8],
+      "ils": [true, true]
+    },
+    {
+      "name": ["11", "29"],
+      "end": [["N38d45m35.83", "W90d24m35.55", "618ft"], ["N38d44m48.46", "W90d22m59.39", "556ft"]],
+      "delay": [15, 6],
+      "ils": [true, true]
+    },
+    {
+      "name": ["12L", "30R"],
+      "end": [["N38d45m06.41", "W90d21m58.66", "528ft"], ["N38d44m18.99", "W90d20m22.51", "605ft"]],
+      "delay": [7, 7],
+      "ils": [true, true]
+    },
+    {
+      "name": ["12R", "30L"],
+      "end": [["N38d45m14.05", "W90d22m44.97", "542ft"], ["N38d44m16.01", "W90d20m47.27", "586ft"]],
+      "delay": [5, 3],
+      "ils": [true, true]
+    }
+  ],
+  "sids": {
```

Comment:
none of these sids have any `body` segments.  there needs to be at least one fix in the `body` section of each sid.